freenet-ext/build.xml

```diff
@@ -251,22 +256,27 @@
 
 	<macrodef name="lib-report">
 		<attribute name="lib" />
-		<sequential>
-			<if>
-				<istrue value="${extlib.@{lib}.suppress}"/>
-				<then><echo message="suppress @{lib} is ON"/></then>
-			</if>
+		<sequential if:true="${extlib.@{lib}.suppress}">
+			<echo message="suppress @{lib} is ON"/>
+		</sequential>
+		<sequential unless:true="${extlib.@{lib}.suppress}">
+			<!-- To preserve the old behaviour of this macro, I've commented this out.
+			<echo message="suppress @{lib} is NOT ON"/>
```

Comment:
Understood and agreed. Unless you have reason to do otherwise, my preference would be to forego a `NOT ON` echo for the reasons mentioned above.